grant-ssh-access-forced-command.py

```diff
@@ -156,6 +156,20 @@ def write_welcome_message(home_dir: Path):
     subprocess.check_call(['sudo', 'sh', '-c', command])
 
 
+def is_dir_writable(home_dir: Path):
+    filepath = home_dir / '.testfile'
+    try:
+        os.remove(filepath)
+        filehandle = open(filepath, 'w')
```

Comment:
in Python you can do "with open(...) as fd:" to always automatically close the file handle